documentation/usersguide/whatsnew.md

```diff
@@ -12,6 +12,17 @@ title: What's New
 #### Upcoming next release
 See upcoming changes from the next version on [github](https://github.com/naemon/naemon-core/blob/master/NEWS)
 
+#### 1.2.4 - 03 Mar 2021
+##### Changes
+  * improve config check when having lots of servicegroups
+  * change header file location in deb/rpm packages
+
+##### Bugfixes
+  * fix svc in check_command neb broker call
```

Comment:
Ahh, I missed this in the core repo, but this should probably be:

"Fix check_command in svc neb broker call"

Reply:
https://github.com/naemon/naemon-core/pull/358